lib/tasks/jenkins.rake

```diff
@@ -11,10 +10,9 @@ begin
 
     namespace :setup do
       task :pre_ci do
-        ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
-        gem 'ci_reporter'
+        ENV['USE_MEAN_TIME_REPORTER'] = '0'
```

Comment:
this can be dropped, once https://github.com/Katello/katello/pull/9826 is merged

Reply:
Can we have a comment for that, or do we believe oursleves to remember?

Reply:
as the author of both, I'll try to remember? ;)